install/Knative-with-IKS.md

```diff
@@ -167,7 +167,7 @@ You can install the Knative Serving and Build components together, or Build on i
 1. Run the `kubectl apply` command to install
    [Knative Build](https://github.com/knative/build) and its dependencies:
     ```bash
-    kubectl apply -f https://raw.githubusercontent.com/knative/serving/v0.1.0/third_party/config/build/release.yaml
+    kubectl apply -f https://raw.githubusercontent.com/knative/serving/v0.1.1/third_party/config/build/release.yaml
```

Comment:
Should this point to the vendored serving copy, or the separate build release (if it exists)?

Reply:
I think I'd rather make more substantive changes in a broader update around decoupling, if that's alright?